docs/deployment/landing_page.rst

```diff
@@ -0,0 +1,251 @@
+.. _Landing Page:
+
+Landing Page
+============
+
+URL and location
+----------------
+
+Ideally you would not use a separate subdomain, but would use a path at
+your top-level domain, e.g. organization.com/securedrop. This is because
+TLS does not encrypt the hostname, so a SecureDrop user whose connection
+is being monitored would be trivially discovered.
+
+If the landing page is deployed on the same domain as another site, you
+might consider having some specific configuration (such as the security
+headers below) apply only to the /securedrop request URI. This can be done
+in Apache by the encapsulating these settings within a
+`<Location> <https://httpd.apache.org/docs/2.4/mod/core.html#location>`__
+block, which can be defined similarly in nginx by using the
+`location {} <http://nginx.org/en/docs/http/ngx_http_core_module.html#location>`__
+directive.
+
+HTTPS only (no mixed content)
+-----------------------------
+
+Most news organizations, `in fact almost
```

Comment:
things have improved somewhat since authorship, I'd update it to say "Many news organizations do not use HTTPS" and link to https://securethe.news/ instead

Reply:
Thanks @ageis - note this PR is scoped to just breaking up this long document as described in #2339 in preparation for a larger revision of the deployment guidelines, will tackle updates in a followup.